runtime/service.go

```diff
@@ -646,6 +647,21 @@ func (s *service) StopVM(requestCtx context.Context, request *proto.StopVMReques
 		timeout = time.Duration(request.TimeoutSeconds) * time.Second
 	}
 
+	info, err := s.machine.DescribeInstanceInfo(requestCtx)
+	if err != nil {
+		errors.Wrapf(err, "failed to get instance info %v", info)
+		return nil, err
+	}
+
+	if *info.State == models.InstanceInfoStatePaused {
+		s.logger.Debugf("Instance is paused state: %v, force shutdown in progress", *info.State)
```

Comment:
Wouldn't it print `Instance is paused state: Paused, force shutdown in progress`? Does it have to say "Paused" twice?